frontend/public/components/cluster-service-broker.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-undef */
+
+import * as React from 'react';
+import { ColHead, DetailsPage, List, ListHeader, ListPage, ResourceRow } from './factory';
+import { Cog, SectionHeading, detailsPage, navFactory, ResourceLink, ResourceCog, ResourceSummary, StatusWithIcon, Timestamp } from './utils';
+// eslint-disable-next-line no-unused-vars
+import { K8sResourceKind, K8sResourceKindReference } from '../module/k8s';
+import {Conditions} from './conditions';
+import { ClusterServiceClassPage } from './cluster-service-class';
+
+const ClusterServiceBrokerReference: K8sResourceKindReference = 'ClusterServiceBroker';
+const menuActions = Cog.factory.common;
+
+const ClusterServiceBrokerHeader: React.SFC<ClusterServiceBrokerHeaderProps> = props => <ListHeader>
+  <ColHead {...props} className="col-md-3 col-sm-3 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-md-3 col-sm-3 col-xs-6" sortFunc="serviceCatalogStatus">Status</ColHead>
+  <ColHead {...props} className="col-md-3 col-sm-3 hidden-xs" sortField="spec.relistBehavior">Relist Behavior</ColHead>
+  <ColHead {...props} className="col-md-3 col-sm-3 hidden-xs" sortField="status.lastCatalogRetrievalTime">Last Retrieved</ColHead>
```

Comment:
The `col-md-*` declarations shouldn't be needed if they match the `col-sm-*` ones exactly